modules/extensions-spring-support/src/test/java/org/mule/module/extension/internal/connector/PetStoreThreadingProfileTestCase.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.connector;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+import org.mule.functional.junit4.ExtensionFunctionalTestCase;
+import org.mule.module.extension.internal.runtime.connector.petstore.PetStoreConnector;
+import org.mule.module.extension.internal.util.ExtensionsTestUtils;
+
+import org.junit.Test;
+
+public class PetStoreThreadingProfileTestCase extends ExtensionFunctionalTestCase
+{
+
+    @Override
+    protected String getConfigFile()
+    {
+        return "petstore-threading-profile.xml";
+    }
+
+    @Override
+    protected Class<?>[] getAnnotatedExtensionClasses()
+    {
+        return new Class<?>[] {PetStoreConnector.class};
+    }
+
+    @Test
+    public void customThreadingProfile() throws Exception
```

Comment:
you also added a threading profile on the connectin provider... where's the assertion for that?